exercises/alphametics/description.md

```diff
@@ -2,8 +2,7 @@
 
 Write a function to solve alphametics puzzles.
 
-[Alphametics](https://en.wikipedia.org/wiki/Alphametics) is a puzzle where
-letters in words are replaced with numbers.
+[Alphametics][] is a puzzle where letters in words are replaced with numbers.
```

Comment:
Should we use this form with the empty brackets?
If it matters, it would be more consistent to use the same form everywhere, i.e. `[Some thing][some-thing]`

Reply:
I used the empty form since it was already used in places, but I actually prefer the form that you suggest. I will go ahead and make that change!